queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-01-01';
```

Comment:
- [ ] Please make sure to select all animals that have a birth date between the years `2016` and `2019`, meaning that if they had a birth date in the year 2019 (until `2019/31/12`) they should be selected.

Reply:
Hello @ShadyShawkat 👋🏼 , 

Thank you for taking out time to review and recommend changes to this PR. The recommended changes will be implemented immediately. 👍🏼